txeventq/kafka-connector-example/sql/setupTeq.sql

```diff
@@ -0,0 +1,20 @@
+set serveroutput on
+
+alter pluggable database all close immediate;
```

Comment:
Why are we closing all pluggable databases? What if there are 50 more?
Let's just close the one that we are about to recreate